src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried ? (`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`)
```

Comment:
Let's create a separate variable for person partner

`const partner = person.sex === 'm' ? 'wife' : 'husband'`

And use this variable here